clients/lsp-fsharp.el

```diff
@@ -179,7 +179,9 @@ available, else the globally installed tool."
 
 (defun lsp-fsharp--fsac-cmd ()
   "The location of fsautocomplete executable."
-  (or (expand-file-name "fsautocomplete" lsp-fsharp-server-install-dir)
+  (or (-let [maybe-local-executable (expand-file-name "fsautocomplete" lsp-fsharp-server-install-dir)]
+        (when (f-exists-p maybe-local-executable)
+          maybe-local-executable))
       (executable-find "fsautocomplete")
       (f-join (or (getenv "USERPROFILE") (getenv "HOME"))
```

Comment:
on related note, I just found out that we have `lsp-fsharp-use-dotnet-tool-for-fsac` var that we're not consider here.. this line should be guarded, I think with:
```
(when lsp-fsharp-use-dotnet-tool-for-fsac
    (f-join (or ...
```

but I can do it myself later..